pyfarm/agent/etc/agent.yml

```diff
@@ -63,6 +70,7 @@ agent_api_port: 50000
 # The location where the the agent should save its own
 # logging output to.
 agent_log: $agent_logs_root/agent.log
+agent_log: $agent_logs_root/supervisor.log
```

Comment:
The second line was supposed to start with ```supervisor_log```.

> I assume we'd want either a combined log file or a separate path for each.

Two processes writing to the same file simultaneously could be hairy, and AFAIK, won't even work on Windows.